datasets.py

```diff
@@ -255,7 +255,7 @@ def from_path(self, file_path):
 
         #Note: bbox digits should be: x, y, width, height. all numbers are scaled to be between 0 and 1
         person_bbox = None # optional
-        gender = None # optional, we have used 0 for male and 1 for female when these labels exist (yes, this order is reversed from self.num_gender_images above)
+        gender = [None] # optional, we have used 0 for male and 1 for female when these labels exist (yes, this order is reversed from self.num_gender_images above)
```

Comment:
please include clear details on the specific format, e.g. if you're doing [label1, label2], [bbox1, bbox2]